manifests/profile/vmhost/prereqs.pp

```diff
@@ -34,7 +34,7 @@
   package { 'qemu':
     ensure => 'installed'
   }
-  
+
   package { 'qemu-kvm':
```

Comment:
This is just to appease the linter; it doesn't otherwise relate to this pull request.